src/libs/mark-unread.js

```diff
@@ -292,7 +294,7 @@ function addCustomAllReadBtn() {
 				<p data-facebox-id="facebox-description">Are you sure you want to mark all unread notifications as read?</p>
 
 				<div class="full-button">
-					<button id="clear-local-notification" class="btn btn-block">Mark all notifications as read</button>
+					<button id="clear-local-notification" class="btn btn-block" autofocus="true">Mark all notifications as read</button>
```

Comment:
`autofocus` is a boolean attribute, so can just be `autofocus="true"` => `autofocus`.